src/NumericalAlgorithms/LinearOperators/PowerMonitors.cpp

```diff
@@ -187,14 +200,14 @@ double convergence_rate(const DataVector& power_monitor,
   // Scale the small factor to be a few orders of magnitude smaller than
   // the max slope delta, as SpEC does. But in case the fit happens to have
   // identically zero errors, make sure eps is still nonzero.
-  const double eps = 1.e-3 * std::max(*max_delta, 1.e-15);
+  const double eps = std::max(*max_delta * 1.e-3, 1.e-15);
   double num = 0.0;
   double denom = 0.0;
-  double denom_this_term = 0.0;
+  double one_over_denom_this_term = 0.0;
   for (size_t i = 0; i < slopes.size(); ++i) {
```

Comment:
Move `one_over_denom_this_term` into the loop. It doesn't need to persist outside each iteration

Reply:
I did this to avoid an extra allocation, but I guess the cost when it's just a double is negligible?

Reply:
There is no cost since it's a `double`